app/views/layouts/mailer.text.erb

```diff
@@ -11,16 +11,14 @@ Archive of Our Own
 <%= yield(:footer_note) %>
 
 <% end %>
-If you've received this message in error, please contact Support at <%= root_url %>support.
-
-The Archive of Our Own is a fan-run and fan-supported archive that relies on your donations: <%= donate_url %>.
+<%= t 'mailer.general.footer.general.text', support_url: root_url+"support", donate_url: donate_url %>
 <% if content_for?(:sent_at) %>
 
-Sent at: <%= yield(:sent_at) %>.
+<%= t 'mailer.general.footer.sentat' %>  <%= yield(:sent_at) %>.
```

Comment:
Ironically, the `sentat` and `sentby` calls in the text.erb have the reverse problem of two spaces between `<%= t ... %>` and `<%= yield(:sent_at) %>` (can you make spaces and tabs visible in your editor to catch this kind of thing before review?) Having said that, interpolation will again solve the problem anyway.

Probably best to incorporate the full stops at the end into the yml string too, and remove those from the erb template as well.